### PR TITLE
[Add] Implementation in Python

### PR DESCRIPTION
**Pull request by shin-k-h**
I wrote it because there was a demand for implementation in python.

### CODE REVIEW

**Comment by shin-k-h** [comment]
Is it better to marge CableBehavior class into Cable class?

File: python/main.py [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+class Resistance(Parts):
+    def __init__(self, ohm):
+        self.ohm = ohm
+
+    def getResistance(self):
+        return self.ohm
+
+class SerialCable(Cable):
+    def __init__(self, c):
```
Comment (k5342) [inline python/main.py]: I think it is preferable for defining a default value as `[]`. And the variable name `c` makes a little bit confused.

File: python/main.py [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    pass
+
+class Resistance(Parts):
+    def __init__(self, ohm):
+        self.ohm = ohm
+
+    def getResistance(self):
+        return self.ohm
+
+class SerialCable(Cable):
```
Comment (k5342) [inline python/main.py:27]: I want `Cable#append` method to modify an existing cable after definition.

File: python/main.py [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+    def __init__(self, c):
+        self.c = c
+
+    def getResistance(self):
+        sum = 0.0
+        for p in self.c:
+            sum += p.getResistance()
+        return sum
+
+class ParalleCable(Cable):
```
Comment (k5342) [inline python/main.py]: typo: Parallel

File: python/main.py [hunk 4]
```diff
@@ -0,0 +1,56 @@
+# !/path/to/bin/of/python
```
Comment (k5342) [inline python/main.py]: It seems unnecessary for Python script. It's not a shell script.

File: python/main.py [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+#↑Fill in as appropriate
+
+from abc import ABCMeta, abstractmethod #抽象クラス用
+
+class PartsBehavior(metaclass=ABCMeta):
+    @abstractmethod
+    def getResistance(self):
+        pass
```
Comment (k5342) [inline python/main.py]: Use `raise NotImplementedError` is preferable on an abstract class rather than `pass`.

**Review by k5342 — CHANGES_REQUESTED** [review]
Thanks for sending PR. I wish some modify to merge this.

**Comment by shin-k-h** [comment]
修正しました

**Comment by k5342** [comment]
LGTM! ありがとうございます！

**Comment by k5342** [comment]
Revert しました #8 すみません．こちらひとつのコミットに squash して再度 PR 投げてもらえますか？

**Comment by shin-k-h** [comment]
了解

**Pull request merged**